releasenotes-builder/src/main/java/com/github/checkstyle/Main.java

```diff
@@ -65,6 +68,8 @@ public static void main(String... args) {
                 final CliOptions cliOptions = cliProcessor.getCliOptions();
                 final Result notesBuilderResult = runGithubNotesBuilder(cliOptions);
                 errorCounter = notesBuilderResult.getErrorMessages().size();
+                errorCounter = validateNotes(errorCounter, cliOptions, notesBuilderResult);
+
```

Comment:
There should be no code changes.
All validation should be in same place as we do validation now

Reply:
Changes Done